front/lib/api/assistant/generation.ts

```diff
@@ -238,7 +238,9 @@ export async function constructPrompt(
 
   // if meta includes the string "{ASSISTANTS_LIST}"
   if (meta.includes("{ASSISTANTS_LIST}")) {
-    const agents = await getAgentConfigurations(auth);
+    if (!auth.user())
```

Comment:
This function is called by API by our slack bot which can call any assistant.

You should use "list" if a `user` is set and "all" otherwise ?

Reply:
:+1: sorry for overlooking this